### ISSUE TITLE
GitGraph view broken in latest insider

### ISSUE THREAD

**Issue by dbaeumer** [microsoft/vscode#103012]
Install Git Graph and try to open it. You get

<img width="432" alt="capture" src="https://user-images.githubusercontent.com/1931590/88055325-169de680-cb5f-11ea-8f05-f960d084a35b.png">

on latest insider.

@mjbvz filing against you since it works in stable and last time I saw this is was an issue with the WebView.

**Comment by tn3rb** [comment]
yup same thing happening for me, in the process of disabling all other extensions to see if that has any affect

**Comment by deepak1556** [comment]
@dbaeumer I am unable to repro the issue on macOS

```
Version: 1.48.0-insider
Commit: eb1ca267d6a478734bd7f3d629ac9b80926ee9d1
Date: 2020-07-21T12:39:31.001Z
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
```

can you open the webview developer tools and see if there are any errors ?

**Comment by mjbvz** [comment]
Seems fixed by ec8606cb49ec839ccdf90c278f44c5bfa3567063 for me
